### ISSUE TITLE
Plugin doesn't work with Bazel 1.0.0rc2

### ISSUE THREAD

**Issue by davido** [bazelbuild/intellij#1166]
After upgrade of Bazel from 0.29.1 to 1.0.0rc2, intellij plugin doesn't work any more. To reproduce, clone recursively [gerrit](https://gerrit.googlesource.com/gerrit) from this commit (b02962142671ff8d933f2d81a3d5aaa6558065c9) and try to import it in IntelliJ.

Bazel Console log in 0.29.1: [1].
Bazel Console log on 1.0.0rc2: [2].

Particularly: the external dependencies are not detected on 1.0.0rc2:

```
[...]
Parsing build outputs...
Total rules: 2139, new/changed: 154, removed: 495
Reading IDE info result...
Updating target map
Loaded 154 aspect files, total size 1080kB
Target map size: 597
Reading jdeps files...
Loaded 0 jdeps files, total size 0kB
Reading package manifests...
Java content entry count: 1
Updating Jar Cache...
Prefetching files...
Refreshing files
Computing directory structure...
Committing project structure...
Workspace has 0 libraries
```

While all is fine on 0.29.1:

```
[...]
Parsing build outputs...
Total rules: 2139, new/changed: 2139, removed: 0
Reading IDE info result...
Updating target map
Loaded 2139 aspect files, total size 3789kB
Target map size: 597
Reading jdeps files...
Loaded 207 jdeps files, total size 354kB
Reading package manifests...
Java content entry count: 22
Updating Jar Cache...
Prefetching files...
Refreshing files
Computing directory structure...
Committing project structure...
Workspace has 287 libraries
```

Environment:

IntelliJ IDEA Ultimate 2019.2.2
Bazel plugin: 2019.08.19.0.5

* [1] http://paste.openstack.org/show/775440
* [2] http://paste.openstack.org/show/775441

**Comment by jin** [comment]
The plugin containing this fix (https://github.com/bazelbuild/intellij/commit/5f03bde0f02f5d1693a4567dc129230420a30372) is expected to be available early next week.

**Comment by davido** [comment]
@jin Any updates on this?

**Comment by ittaiz** [comment]
@davido maybe it should be closed since #1213 is closed
